### PR TITLE
Feature/sdk 31 sdk 35 behaviosec info

### PR DESCRIPTION
**Pull request by gadulin**

### CODE REVIEW

File: kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java [hunk 1]
```diff
@@ -591,6 +609,168 @@ public JSONObject getUniques(String deviceId, Map<String, String> additionalPara
...
+	 * @param timing
+	 *            data gathered from a BehavioSec collection
+	 * @param uniq
+	 *            customer identifier
+	 * @return A JSONObject containing the response.
+	 * @throws AccessException
+	 *             Thrown if any of the parameter values are invalid or there
+	 *             was a problem getting a response.
+	 */
+	public void setBehaviorData(String host, String environment, String session, String timing, String uniq)
```
Comment (boyankukushev) [inline kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java]: `setBehavio` -- no `r`

File: kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java [hunk 2]
```diff
@@ -591,6 +609,168 @@ public JSONObject getUniques(String deviceId, Map<String, String> additionalPara
...
+	 * @param uniq
+	 *            customer identifier
+	 * @param additionalParameters
+	 *            Additional parameters to send to server.
+	 * @return A JSONObject containing the response.
+	 * @throws AccessException
+	 *             Thrown if any of the parameter values are invalid or there
+	 *             was a problem getting a response.
+	 */
+	private void setBehaviorData(String host, String environment, String session, String timing, String uniq,
```
Comment (boyankukushev) [inline kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java]: `behavio`

File: kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java [hunk 3]
```diff
@@ -591,6 +609,168 @@ public JSONObject getUniques(String deviceId, Map<String, String> additionalPara
...
+		String response = this.postRequest(infoEndpoint, parameters);
+		logger.debug("request elapsed time = " + (System.currentTimeMillis() - startTime) + ", response = " + response);
+		if (response != null) {
+			return processJSONEntity(response);
+		}
+
+		return null;
+	}
+
+	private void verifyInfoParams(int infoFlag, String uniq, String username, String password) throws AccessException {
```
Comment (boyankukushev) [inline kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java]: The flag checks in this method are hard to read and maintain. Please look at the dotnet sdk implementation, Galin did very good job in summarizing those.

File: kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java [hunk 4]
```diff
@@ -729,6 +933,7 @@ private String postRequest(String urlString, List<NameValuePair> values) throws
 			CloseableHttpClient client = getHttpClient();
 			HttpPost request = getHttpPost(urlString);
 			request.addHeader("Authorization", this.getAuthorizationHeader());
+			// request.addHeader("Accept", "application/x-www-form-urlencoded");
```
Comment (boyankukushev) [inline kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java]: Remove this if it's not needed.

File: kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java (hunk 3 above)
Comment (gadulin) [reply kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java]: Done.

**Pull request merged**